README.md

```diff
@@ -65,7 +65,7 @@ site=https://realm.zulipchat.com
 [zterm]
 # Theme can also be set to 'gruvbox' ('blue' & 'light' are older themes needing work)
 theme=default
-# Autohide can also be set to 'no_autohide', to always show the left and right panels
+# Autohide can also be set to 'autohide', to hide the left and right panels by default
```

Comment:
I think this needs to be slightly clearer; perhaps "... to only show the left and right panels when required" ?
(or 'necessary', or 'focused'?)

Reply:
How about - If you want to view only messages, you can set `autohide` as the default setting. Then, left and right panels will only be visible when focused.

Reply:
@amanagr How about the following?
```
# Autohide defaults to 'no_autohide', but can be set to 'autohide' to hide the left & right panels except when focused
```

I think it would be good to get this merged ASAP for new devs or people running from master.

Reply:
Cool, done!

Reply:
The comment text great! LGTM :)